symphony-bdk-core/src/main/java/com/symphony/bdk/core/util/PresentationMLParser.java

```diff
@@ -0,0 +1,48 @@
+package com.symphony.bdk.core.util;
```

Comment:
Related to message, so should be located in `com.symphony.bdk.core.service.message.util`